src/test/java/uk/gov/ons/fwmt/job_service/controller/tm_endpoint/GenericOutgoingWsTest.java

```diff
@@ -16,7 +19,9 @@ public void sendAdapterOutput() {
     //Given
 
     //When
-    JAXBElement<SendMessageResponse> result = genericOutgoingWs.sendAdapterOutput(null);
+    QName qname = new QName("request");
+    JAXBElement<WebServiceAdapterOutputRequest> jaxbElement = new JAXBElement<WebServiceAdapterOutputRequest>(qname, WebServiceAdapterOutputRequest.class, new WebServiceAdapterOutputRequest());
+    JAXBElement<SendMessageResponse> result = genericOutgoingWs.request(jaxbElement);
 
     //Then
     assertNotNull(result);
```

Comment:
Can the assertion be changed to something more useful now?

Reply:
No, at this point the webservice has no functionality. Just getting a response back is what is required for Pen Testing